enterprise_subsidy/apps/api/v1/views/subsidy.py

```diff
@@ -371,3 +374,31 @@ def partial_update(self, request, *args, **kwargs):
         """
         kwargs['partial'] = True
         return self.update(request, *args, **kwargs)
+
+    @extend_schema(
+        tags=['subsidy'],
+        request=SubsidyLearnerAggregateRequestSerializer,
+        parameters=[Parameters.ENTERPRISE_SUBSIDY_ACCESS_POLICY_UUID],
+        responses={
+            200: SubsidyLearnerAggregateSerializer,
+            403: exceptions.PermissionDenied,
```

Comment:
[nit] These should be in numerical order

Reply:
oops hahaha nice catch